### PR TITLE
Filenames inherit chart title, disable table button added.

### PR DESCRIPTION
**Pull request by rob-duplock**

### CODE REVIEW

File: js/jquery.dvfTables.js [hunk 1]
```diff
@@ -97,6 +97,10 @@
      */
     addToggleButton: function () {
 
+      if (this.options.disable_table) {
```
Comment (rob3000) [inline js/jquery.dvfTables.js]: @rob-duplock this should be camelcase

File: src/Plugin/Visualisation/Style/TableVisualisationStyleBase.php [hunk 2]
```diff
@@ -41,9 +41,12 @@ protected function buildTable(array $records) {
    *   An array of table build settings.
    */
   protected function tableBuildSettings(array $records) {
+    $config = $this->getConfiguration();
+
     return [
       'data' => $this->getTableRows($records),
       'columns' => $this->getTableHeader(),
+      'disable_table' => $config['chart']['disable_table'],
```
Comment (rob3000) [inline src/Plugin/Visualisation/Style/TableVisualisationStyleBase.php]: Should be camel case here right for the js plugin?

**Review by rob3000 — CHANGES_REQUESTED** [review]

File: src/Plugin/Visualisation/Style/TableVisualisationStyleBase.php (hunk 2 above)
Comment (jez500) [reply src/Plugin/Visualisation/Style/TableVisualisationStyleBase.php]: @rob-duplock are all the other configs camel case? camel is prefered but if everything else uses underscores then IMO consistency is more important

Comment (rob-duplock) [reply src/Plugin/Visualisation/Style/TableVisualisationStyleBase.php]: In the interest in consistency with other parts of the module, I've now implemented this as `this.options.table.disable` / `$config['chart']['table']['disable']` which is how its done elsewhere.

Comment (rob-duplock) [reply src/Plugin/Visualisation/Style/TableVisualisationStyleBase.php]: Hey @jez500 could you please give this PR another look? I have implemented fixes as discussed.

**Review by jeremy-doghouse — APPROVED** [review]
Nice work @rob-duplock

**Pull request merged**